.travis.yml

```
@@ -8,7 +8,7 @@ install:
   - go install github.com/golangci/golangci-lint/cmd/golangci-lint
 script:
   - golangci-lint run --enable-all -D errcheck -D lll -D dupl -D gochecknoglobals --deadline 5m
-  - go test -v -coverprofile=coverage.txt -covermode=atomic
+  - go test -v ./... -coverprofile=coverage.txt -covermode=atomic
```

iirc, Go doesn't like mixing flags and positional args, this should probably fix it:

```suggestion
  - go test -v -coverprofile=coverage.txt -covermode=atomic ./...
```